pkg/intcode/intcode.go

```diff
@@ -153,7 +127,8 @@ func (*multiply) run(p *program, arg1, arg2, arg3 int) bool {
 
 func (r *input) run(p *program, arg1, arg2, arg3 int) bool {
 	p.index += 2
-	p.instructions[arg1] = p.input
+	p.instructions[arg1] = p.inputs[0]
+	p.inputs = p.inputs[1:]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [avoid magic numbers like '1', create a named constant for it](https://app.codacy.com/manual/afarbos/aoc/pullRequest?prid=5498712)